torchbenchmark/__init__.py

```diff
@@ -136,6 +136,28 @@ def name(self) -> str:
         return os.path.basename(self.path)
 
 
+class Worker(subprocess_worker.SubprocessWorker):
+    """Run subprocess using taskset if CPU affinity is set.
```

Comment:
This is a LONG docstring! Thanks for the clarification.

Reply:
And about to get a bit longer.